viper.go

```diff
@@ -282,6 +283,15 @@ func EnvKeyReplacer(r StringReplacer) Option {
 	})
 }
 
+func StrictEnvPrefix(in string) Option {
```

Comment:
I'm not sure I like the name `StrictEnvPrefix`.

Something like `RawEnvPrefix` or `BareEnvPrefix` might sound better.

An alternative I can imagine is a separate option called EnvPrefixWithDelimiter (deprecating any current options).